site/about/deployment/deployment-options.qmd

```diff
@@ -0,0 +1,70 @@
+---
+title: "Deployment options"
+date: last-modified
+listing:
+  - id: validmind-deployment-options
+    type: grid
+    max-description-length: 250
+    sort: false
+    grid-columns: 2
+    fields: [title, description]
+    contents:
+      - multi-tenant-cloud.qmd
+      - virtual-private-validmind.qmd
+---
+
+Choose the {{< var vm.product >}} deployment option that best suits your organization, ensuring secure data protection and seamless integration with your existing environment.
+
+## Available options
+
+Our deployment options provide a balance of control and cost-efficiency while integrating seamlessly with your infrastructure. For added flexibility, you can deploy on Amazon Web Services (AWS), Google Cloud Platform (GCP), or Microsoft Azure.
+
+We offer two deployment models:
+
+- **Multi-tenant cloud** — Multiple organizations (tenants) share infrastructure while keeping data isolated, providing cost-efficiency and scalability. VPC support allows secure network connections.
+
+- **Virtual Private ValidMind (VPV)** — A single-tenant architecture where one organization uses a dedicated infrastructure instance, offering greater control, customization, and enhanced security. This option is ideal for high-compliance needs and also supports VPC for secure connectivity.
+
+## Architecture overview
+
+
+![{{< var vm.product >}} architecture overview](validmind-architecture-overview.png){fig-alt="An image showing the ValidMind platform architecture"}
+
+In your own environment, model developers can continue to run models using your existing tools for data science and model development, such as Python, Jupyter Notebooks, and R, accessing data from sources such as Google Cloud Storage, Amazon S3, and Snowflake.
+
+These models are then integrated with the {{< var validmind.developer >}}, which communicates with {{< var vm.product >}} via our API.
+
+The ValidMind Platform provides:
+
+- **Model inventory** — Centralized tracking and organization of models, accessible by developers, validators, and executives.
+
+- **Documentation & validation engine** — Automated testing and documentation, with validation processes, ensuring compliance with regulations and internal policies.
+
+- **Template management** — Allows for easy creation, customization, and reuse of documentation templates.
+
+- **ValidMind dashboard** — A user-friendly interface providing insights, status updates, and governance reporting for model risk.
+
+## Security & data privacy
+
+We ensure data security through strong data isolation, encryption, and role-based access controls.[^1] Personal identifiable information and customer data are not stored in model documentation. For more information, see our data privacy policy.[^2]
+
+## Secure access
```

Comment:
Secure access is provided via private link, and VPCs are just how we isolate individual ValidMind deployments, so it doesn't make much sense in this sentence.

Something like this could work:

Deployments support secure access via private link, giving you full control over your resources without exposing traffic to the public internet. Integration options include:

	•	AWS PrivateLink
	•	Google Cloud Private Service Connect
	•	Azure Private Link

Additionally, all connections are secured with HTTPS/SSL to ensure encrypted communication.

Reply:
Changed verbatim as suggested.